.github/release.yml

```diff
@@ -0,0 +1,22 @@
+# .github/release.yml
+
+changelog:
+  exclude:
+    labels:
+      - Ignore-For-Release
+  categories:
+    - title: Breaking Changes 🚨
+      labels:
+        - Breaking-Change
+    - title: Exciting New Features 🎉
+      labels:
+        - New-Feature
+    - title: Improvements 🌱
+      labels:
+        - Improvement
+    - title: Bug Fixes 🛠
+      labels:
+        - Bug-Fixes
```

Comment:
1. I think the convention is to use lower case labels.
2. Let's use the `bug` label, to make sure we will not confuse between `bug fix` and `bug`.
```suggestion
      - ignore for release
  categories:
    - title: Breaking Changes 🚨
      labels:
        - breaking change
    - title: Exciting New Features 🎉
      labels:
        - new feature
    - title: Improvements 🌱
      labels:
        - improvement
    - title: Bug Fixes 🛠
      labels:
        - bug
```